BoostTestAdapter/Resources.resx

```diff
@@ -0,0 +1,298 @@
+  </data>
+  <data name="Debugging" xml:space="preserve">
+    <value>Debugging:   -&gt; [{0}]</value>
+    <comment>{0} represents a list of test names</comment>
+  </data>
+  <data name="DebuggingPropertiesNotFound" xml:space="preserve">
+    <value>Could not obtain debugging properties for {0}.</value>
+    <comment>{0} represents a source file name</comment>
+  </data>
+  <data name="Discovering" xml:space="preserve">
+    <value>Discovering ({0}):   -&gt; [{1}]</value>
+    <comment>{0} represents the name of a test discoverer class
+{1} represents a list of source file names</comment>
+  </data>
+  <data name="DiscoveryException" xml:space="preserve">
+    <value>Exception caught while discovering tests: {0} ({1})</value>
+    <comment>{0} represents an exception message
+{1} represents an exception HResult</comment>
+  </data>
+  <data name="DiscoveryExceptionFor" xml:space="preserve">
+    <value>Exception caught while discovering tests for {0} ({1} - {2})</value>
+    <comment>{0} represents a source file name
+{1} represents an exception message
+{2} represents and exception HResult</comment>
+  </data>
+  <data name="Exception" xml:space="preserve">
+    <value>Exception: {0} ({1})</value>
+    <comment>{0} represents an exception message
+{1} represents an exception HResult</comment>
+  </data>
+  <data name="Executing" xml:space="preserve">
+    <value>Executing:   -&gt; [{0}]</value>
+    <comment>{0} represents a list of test names</comment>
+  </data>
+  <data name="ExecutorNotFound" xml:space="preserve">
+    <value>No suitable executor found for [{0}].</value>
+    <comment>{0} represents a list of test names</comment>
+  </data>
+  <data name="FailedToAddProcess" xml:space="preserve">
+    <value>Process could not be added to Job Object. Test process may end up orphaned on abrupt closure.</value>
+  </data>
+  <data name="FailedToKillProcess" xml:space="preserve">
+    <value>Unable to kill process {0}. Process may still be running.</value>
+    <comment>{0} represents a process ID</comment>
+  </data>
+  <data name="FileNotFound" xml:space="preserve">
+    <value>File {0} does not exist.</value>
+    <comment>{0} represents a source file name</comment>
+  </data>
+  <data name="FindingChildren" xml:space="preserve">
+    <value>Finding processes spawned by process with Id [{0}]</value>
+    <comment>{0} represents a process ID</comment>
+  </data>
+  <data name="InvalidPropertyFile" xml:space="preserve">
+    <value>Invalid file defining property under {0}. {1}</value>
+    <comment>{0} represents an XML tag name
+{1} represents an exception message</comment>
+  </data>
+  <data name="KilledProcess" xml:space="preserve">
+    <value>Successfully killed process {0}.</value>
+    <comment>{0} represents a process ID</comment>
+  </data>
+  <data name="ListContentsFile" xml:space="preserve">
+    <value>list_content file: {0}</value>
+    <comment>{0} represents a file path</comment>
```

Comment:
Please indicate in the comment that "list_content" is a Boost.Test command line parameter that shall not be localized. Same below.